h/panels/footer.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+
+"""The footer displayed at the bottom of most pages."""
+
+from __future__ import unicode_literals
+
+from pyramid_layout.panel import panel_config
+
+
+@panel_config(name='footer', renderer='h:templates/panels/footer.html.jinja2')
+def footer(context, request):
```

Comment:
It's probably not going to have any logic I guess, but I just did it as a panel for consistency with the nav bar. Could be changed to an include though (the I wouldn't need to add a Python test to keep codecov happy!)